src/main/io/osd.c

```diff
@@ -811,60 +812,64 @@ static void osdUpdateStats(void)
 
 static void osdShowStats(void)
 {
+    const char * disarmReasonStr[DISARM_REASON_COUNT] = { "UNKNOWN", "TIMEOUT", "STICKS", "SWITCH", "SWITCH", "KILLSW", "FAILSAFE", "NAV SYS" };
     uint8_t top = 2;
-    const uint8_t statValuesX = 21;
+    const uint8_t statValuesX = 20;
     char buff[10];
 
     displayClearScreen(osdDisplayPort);
-    displayWrite(osdDisplayPort, 2, top++, "  --- STATS ---");
+    displayWrite(osdDisplayPort, 1, top++, "  --- STATS ---");
```

Comment:
Now that you're changing this, maybe we can take the opportunity to use a constant? e.g. `statNamesX`

Reply:
Done

Reply:
Thanks!